src/Illuminate/Database/Eloquent/Relations/Concerns/InteractsWithPivotTable.php

```diff
@@ -12,7 +12,7 @@ trait InteractsWithPivotTable
     /**
      * The cached copy of the currently attached pivot models.
      *
-     * @var Collection
+     * @var \Illuminate\Support\Collection
```

Comment:
This doc is actually incorrect. It needs to be `\Illuminate\Database\Eloquent\Collection`.

Reply:
Right, fixed it.